docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -37,6 +37,13 @@ MSCK REPAIR TABLE table_identifier
 
     **Syntax:** `[ database_name. ] table_name`
 
+* **`[[ADD|DROP|SYNC] PARTITIONS]`**
+
+    * If the option is not specified, `MSCK REPAIR TABLE` adds partitions to the Hive external catalog only.
```

Comment:
`the option` sounds a little mismatched because this is a SQL syntax. Are you mentioned the optional syntaxes like this in our SQL docs?

Reply:
I will remove `the option`, I found it in another command: http://spark.apache.org/docs/latest/sql-ref-syntax-aux-analyze-table.html
```
If no analyze option is specified, ANALYZE TABLE collects ...
```
but docs for other commands say `If specified, ...`